cmake/toolchain/arcmwdt/generic.cmake

```diff
@@ -2,17 +2,17 @@
 
 zephyr_get(ARCMWDT_TOOLCHAIN_PATH)
 
-if(NOT DEFINED ARCMWDT_TOOLCHAIN_PATH AND DEFINED ENV{METAWARE_ROOT})
-  message(WARNING "ARCMWDT_TOOLCHAIN_PATH is not set, use default toolchain from METAWARE_ROOT")
-  set(METAWARE_ROOT $ENV{METAWARE_ROOT})
+set(METAWARE_ROOT $ENV{METAWARE_ROOT})
+if(NOT DEFINED ARCMWDT_TOOLCHAIN_PATH AND DEFINED METAWARE_ROOT)
+  message(STATUS "ARCMWDT_TOOLCHAIN_PATH is not set, use default toolchain from METAWARE_ROOT: '${METAWARE_ROOT}'")
   if(NOT EXISTS ${METAWARE_ROOT})
     message(FATAL_ERROR "Nothing found at METAWARE_ROOT: '${METAWARE_ROOT}'")
   endif()
   cmake_path(GET METAWARE_ROOT PARENT_PATH ARCMWDT_TOOLCHAIN_PATH)
+elseif(NOT DEFINED ARCMWDT_TOOLCHAIN_PATH AND NOT DEFINED METAWARE_ROOT)
+  message(FATAL_ERROR "No MWDT toolchain specified: neither ARCMWDT_TOOLCHAIN_PATH nor METAWARE_ROOT defined")
```

Comment:
minor nit.
Should it be highlighted that `METAWARE_ROOT` can only be defined as environment setting ?

(ARCMWDT_TOOLCHAIN_PATH can be defined both as CMake var and in env.)

Reply:
Merging, guess this can go as a followup.